ARSIS-Telemetry-GroundControl/groundcontrol-api/app/event.py

```diff
@@ -3,9 +3,10 @@
 
 class Event:
     @staticmethod
-    def create_event(event_type, data):
+    def create_event(event_type, data, upsert_key=""):
         return {
             "type": event_type,
-            "time": time.time_ns(),
```

Comment:
Will the key "label" hold the time? Will "label" only be of `float` type or will it also be string? I think it will be easier if we can avoid trying to implement a key with multiple types for deserialization in HMD.

Reply:
yeah to avoid that I think we should go with a `time` and a `label` variable